README.md

```diff
@@ -29,7 +29,7 @@ pow's uninstall script. Read more details in [the pow manual](http://pow.cx/manu
 
 #### .dev domain
 
-Install the dev-tld-resolver (https://github.com/puma/dev-tld-resolver) to make domains resolve.
+Install the [dev-tld-resolver](https://github.com/puma/dev-tld-resolver) to make domains resolve.
```

Comment:
I prefer this style of MD links, but I'm happy to revert this change if it's not adding anything.